Makefile

```diff
@@ -16,4 +16,5 @@ test: image
 citest:
 	docker run --rm \
 		--workdir /usr/src/app \
+		--volume /tmp:/tmp \
```

Comment:
Build wasn't happy without it:

```
  1) validateConfig returns false if no files exist:
     Uncaught AssertionError: expected undefined to equal true
      at test/validate_config_test.js:15:44
      at node_modules/temp/lib/temp.js:252:7
      at FSReqWrap.oncomplete (fs.js:123:15)

  2) validateConfig Uncaught error outside test suite:
     Uncaught TypeError: Cannot read property 'currentRetry' of undefined
```

https://circleci.com/gh/codeclimate/codeclimate-eslint/476

Happy to add a note in the commit description. The `temp` module seemed happier with the volume mount.